src/ucp/core/ucp_ep.c

```diff
@@ -632,24 +634,23 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
 
                 if (context->config.ext.zcopy_thresh == UCS_CONFIG_MEMUNITS_AUTO) {
                     /* auto */
-                    zcopy_thresh = md_attr->reg_cost.overhead / (
-                                            (1.0 / context->config.ext.bcopy_bw) -
-                                            (1.0 / iface_attr->bandwidth) -
-                                            md_attr->reg_cost.growth);
-                    if (zcopy_thresh < 0) {
-                        config->am.zcopy_thresh      = SIZE_MAX;
-                        config->am.sync_zcopy_thresh = -1;
-                    } else {
-                        config->am.zcopy_thresh      = zcopy_thresh;
-                        config->am.sync_zcopy_thresh = zcopy_thresh;
+                    config->am.zcopy_thresh[0]      = UCS_CONFIG_MEMUNITS_AUTO;
```

Comment:
@yosefe This is indicator for the user defined threshold option. Without such indicator we can't decide calculate the threshold (later in `ucp_tag_req_start` in case of `iovcnt > UCP_MAX_IOV`) or use `zcopy_thresh[1]` because it was set by the user.
In case of user threshold input, we can populate the same variable into all zcopy_thresh[] cells but we have no flag in the places where we will use it.